### DISCUSSION TITLE
Changing destination in Intersection environment

### DISCUSSION THREAD

**Discussion by nedaasha** [eleurent/highway-env#407] [Q&A]
Hello,
I want to change the destination of the vehicles in the multi-agent setting of the intersection environment according to the following code:

env.configure({"controlled_vehicles": 3})

env.configure({
  "action": {
    "type": "MultiAgentAction",
    "action_config": {
      "type": "ContinuousAction",
      'lateral': False,
      'longitudinal': True
    }
  }
})

env.configure({
  "observation": {
    "type": "MultiAgentObservation",
    "observation_config": {
      "type": "Kinematics",
    },
  }
  
})
env.configure({"destination": "o2"})

however, the destinations remain unchanged and equal to the default value which is 'o1'. Is there any other way to modify the destination of the vehicles?

**Comment by eleurent** [comment]
(don't forget to reset the environment after setting the configuration)
The "destination" config only controls the destination of the controlled vehicles.
For other vehicles, it is chosen randomly here:

https://github.com/eleurent/highway-env/blob/d33b57ae73c233e59c86180f1d1d460964b026a6/highway_env/envs/intersection_env.py#L239

You may edit this code to set the destination to whatever you like.
